lib/UI/global.dart

```diff
@@ -45,7 +46,7 @@ TextStyle mediumTitleStyleWhite = TextStyle(
     fontSize: 18);
 TextStyle setHintStyle = TextStyle(
   letterSpacing: 1,
-  color: Colors.white.withOpacity(0.2),
+  color: Colors.white.withOpacity(1),

```

Comment:
Is there a reason this changed? And why not just use Colors.white seeing as the opacity is 1.

Reply:
Good catch. It should be 0.2 not 1

Reply:
Done, commited.